app/views/gitter/chat_button.erb

```diff
@@ -0,0 +1,17 @@
+<% if control.language.downcase == 'ruby' %>
```

Comment:
We should be checking whether or not we have a value for `gitter` in the track data, rather than checking for Ruby explicitly.